bin/sqlline.py

```diff
@@ -123,8 +126,11 @@ def kill_child():
     '" -Dlog4j2.configurationFile=file:' + os.path.join(phoenix_utils.current_dir, "log4j2.properties") + \
     disable_jna + \
     " sqlline.SqlLine -d org.apache.phoenix.jdbc.PhoenixDriver" + \
-    " -u jdbc:phoenix:" + phoenix_utils.shell_quote([zookeeper]) + \
-    " -n none -p none --color=" + colorSetting + " --fastConnect=" + tryDecode(args.fastconnect) + \
-    " --verbose=" + tryDecode(args.verbose) + " --incremental=false --isolation=TRANSACTION_READ_COMMITTED " + sqlfile
+    (not args.noconnect and " -u jdbc:phoenix:" + phoenix_utils.shell_quote([zookeeper]) or "") + \
```

Comment:
same here, all changes in this file can be removed

Reply:
I would like to include the changes in the sqlline, unless there is a problem with the approach. I can also raise it as a separate PR if that is preferred.

Reply:
yeah i think separate Jira would be better to track it and others can also review, i am not actually sure whether we can make the change or not because at least we don't use this script as is in production env

Reply:
I now moved the change into [PHOENIX-7055](https://issues.apache.org/jira/browse/PHOENIX-7055) and created a separate PR #1695. Please take a look.